src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingBeekeeper.java

```diff
@@ -231,7 +231,7 @@ public boolean shouldHarvestHoneycombs()
      */
     public int getMaximumHives()
     {
-        return (int) Math.floor(Math.pow(2, getBuildingLevel() - 1));
+        return (int) Math.floor(Math.pow(2, getBuildingLevel()));
```

Comment:
return 1 << getBuildingLevel();

Reply:
Yep. But it not really relevant, but a good idea todo anyway.

Reply:
I'd say no. While it is quicker. It makes this code much less readable about what we want to do.

Reply:
We also have this everywhere like this exactly already as code.

Reply:
I have to agree with @Raycoms. The shift left is faster, but far less readable. If we want to switch to it, then I think that should be a separate change that hits all of the code to keep it consistent. There are 13 locations in the code today to be reviewed.

Reply:
I only pointed out the left shift because it avoid the possibilities for rounding issues - esp when using Math.floor() instead of Math.round().

Reply:
Yeah, I backed out the change at @Minecraftschurli's request, and then realized a few minutes ago that what I was seeing was likely due to the use of floor. I'm doing a bit of testing, and will likely remove the floor. None of the other 12 locations use floor.

Reply:
Ok, floor gone.